### DISCUSSION TITLE
SelenideElement.parent() how does it work?

### DISCUSSION THREAD

**Discussion by Elvintest** [selenide/selenide#2219] [Q&A] [answered]
Hello to everyone i was wondering to know how the function parent() is composed and written inside Selenide library considering it can get you a parent by css. Does it somehow transform element by xpath and then get a parrent or some js functions work? I would be grateful if someone opens this insite for me:)

**Comment by rosolko** [comment, marked as answer]
Yes, command uses xpath inside: https://github.com/selenide/selenide/blob/main/src/main/java/com/codeborne/selenide/commands/GetParent.java#L29

**Comment by Elvintest** [comment]
Thanks for the answers!
